Automated docker imagess using github workflows

Any thoughts on adding automated docker images to be built using github workflows so users dont need to clone and build the image themselves?

No seems like something is wrong with the docker image,
```
node:internal/modules/cjs/loader:1148
  throw err;
  ^

Error: Cannot find module '/app/index.js'
    at Module._resolveFilename (node:internal/modules/cjs/loader:1145:15)
    at Module._load (node:internal/modules/cjs/loader:986:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
    at node:internal/main/run_main_module:28:49 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

Node.js v20.16.0
```

> No seems like something is wrong with the docker image,
> 
> ```
> node:internal/modules/cjs/loader:1148
>   throw err;
>   ^
> 
> Error: Cannot find module '/app/index.js'
>     at Module._resolveFilename (node:internal/modules/cjs/loader:1145:15)
>     at Module._load (node:internal/modules/cjs/loader:986:27)
>     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:174:12)
>     at node:internal/main/run_main_module:28:49 {
>   code: 'MODULE_NOT_FOUND',
>   requireStack: []
> }
> 
> Node.js v20.16.0
> ```

Hmm, thats weird. I'll look into this

--- a/Dockerfile
+++ b/Dockerfile
@@ -5,7 +5,7 @@ WORKDIR /app
 COPY package.json .
 COPY package-lock.json .
 
-RUN npm ci
+RUN npm install
 
 COPY . .
 RUN npm run tsc

